app/assets/sass/patterns/_task-list.scss

```diff
@@ -1,72 +1,66 @@
 // Task list pattern
 
-// Override column width for tablet and up
-.app-column-minimum {
-  @include govuk-media-query($from: tablet) {
-    min-width: 600px;
-  }
-}
-
-// Spacing to the left of the task list
-$task-list-indent: 35px;
-
 .app-task-list {
-  list-style: none;
-  padding: 0;
-  margin-top: $govuk-gutter;
+  list-style-type: none;
+  padding-left: 0;
+  margin-top: 0;
+  margin-bottom: 0;
   @include govuk-media-query($from: tablet) {
-    margin-top: ($govuk-gutter * 2);
+    min-width: 550px;
   }
 }
 
 .app-task-list__section {
   display: table;
-
-  @include govuk-font(24, $weight: bold);
-  margin: 0;
-  padding-bottom: ($govuk-gutter / 6);
+  @include govuk-font($size: 24);
+  @include govuk-typography-weight-bold;
```

Comment:
as discussed on slack this seems more complicated than the original approach:

`@include govuk-font(24, $weight: bold);`